addons/l10n_cz/data/tax_report.xml

```diff
@@ -0,0 +1,906 @@
+                            </record>
+                        </field>
+                    </record>
+                    <record id='l10n_cz_vat_declaration_line_47' model='account.report.line'>
+                        <field name='name'>Value of acquired assets as defined in Section 4(4)(d) and (e) - line 47</field>
+                        <field name='code'>R47</field>
+                        <field name='expression_ids'>
+                            <record id='l10n_cz_vat_declaration_line_47_tax_base' model='account.report.expression'>
+                                <field name='label'>tax_base</field>
+                                <field name='engine'>tax_tags</field>
+                                <field name='formula'>VAT 47 Base</field>
+                                <field name='date_scope'>strict_range</field>
+                            </record>
+                            <record id='l10n_cz_vat_declaration_line_47_in_full_amount' model='account.report.expression'>
+                                <field name='label'>in_full_amount</field>
+                                <field name='engine'>tax_tags</field>
+                                <field name='formula'>VAT 47 Total</field>
+                                <field name='date_scope'>strict_range</field>
+                            </record>
+                            <record id='l10n_cz_vat_declaration_line_47_reduced_claim' model='account.report.expression'>
+                                <field name='label'>reduced_claim</field>
+                                <field name='engine'>external</field>
+                                <field name='formula'>sum</field>
+                                <field name='subformula'>editable;rounding=2</field>
+                                <field name='date_scope'>strict_range</field>
+                            </record>
+                        </field>
+                    </record>
+                </field>
+            </record>
+            <record id='l10n_cz_vat_declaration_line_48' model='account.report.line'>
+                <field name='name'>V. Reduction of tax deduction entitlement</field>
+                <field name='hierarchy_level'>0</field>
+                <field name='children_ids'>
+                    <record id='l10n_cz_vat_declaration_line_49' model='account.report.line'>
+                        <field name='name'>Transactions exempt from tax without the right to tax deduction - line 50</field>
+                        <field name='code'>R50</field>
+                        <field name='expression_ids'>
+                            <record id='l10n_cz_vat_declaration_line_49_value' model='account.report.expression'>
+                                <field name='label'>value</field>
+                                <field name='engine'>tax_tags</field>
+                                <field name='formula'>VAT 50</field>
```

Comment:
Just wondering we have multiple element with formula that are not present anywhere, normal ?

Reply:
Seems abnormal in my opinion, but I was blindly following the specs from the partner provided excel sheet https://docs.google.com/spreadsheets/d/1xqDqpLwY5nCyNA5WuzLSPdR52Jq6f8IPxU4S3Bh1Nw4/edit?pli=1&gid=1709389226#gid=1709389226
Not sure if I should amend this or not tbh